docs/windows.md

```diff
@@ -0,0 +1,192 @@
+# Windows Support
+
+Starting with the release version [0.10.0](https://github.com/cloud-hypervisor/cloud-hypervisor/releases/tag/v0.10.0), Cloud Hypervisor supports Windows guests.
+
+Requirements
```

Comment:
I'd suggest `Requirements` to be highlighted by making it bold. `__Requirements__`

Reply:
Ack, done.